python/tests/test_streaming_job_scheduling.py

```diff
@@ -20,19 +20,22 @@
 @pytest.fixture
 def feast_client():
     c = FeastClient(job_service_pause_between_jobs=0)
-    c.list_projects = Mock(return_value=["default"])
+    c.list_projects = Mock(return_value=["default", "ride", "invalid_project"])
     c.list_feature_tables = Mock()
 
     yield c
 
 
 @pytest.fixture
 def spark_client(feast_client):
-    c = Client(feast_client)
-    c.list_jobs = Mock()
-    c.start_stream_to_online_ingestion = Mock()
+    with patch("feast_spark.client.Client._job_service") as job_service:
```

Comment:
could we pass it as regular client configuration and not mock?
eg `Client(whitelisted_projects=...)`